### PR TITLE
LPS-181709 Review and clean CSS

### PR DESCRIPTION
**Pull request by BarbaraCabrera**

### CODE REVIEW

**Comment by liferay-echo** [comment]
Thank you for submitting this PR. 👏

Make sure it follows the [Echo Pull Request Template](https://gist.github.com/liferay-echo/5a4455285ff1800f9da146816d85e67d#file-pull-request-template-md) if you haven't already done so. 

Learn more about it [here](https://liferay.atlassian.net/wiki/spaces/ENGECHO/pages/2027455014/Echo+Pull+Request+Etiquette).

File: modules/apps/segments/segments-web/src/main/resources/META-INF/resources/css/_contributor_builder.scss [hunk 1]
```diff
@@ -9,31 +9,15 @@
...
-			overflow: auto;
-			position: absolute;
-			width: 100%;
 
 			.container-fluid {
-				height: 100%;
-				padding: 0;
 				transition: all 0.5s ease;
 
 				.content-wrapper {
```
Comment (sandrodw3) [inline modules/apps/segments/segments-web/src/main/resources/META-INF/resources/css/_contributor_builder.scss:23]: We have two rules for `.panel-header-link` inside this, we should merge them, and I think we can also change some padding rules for classes

File: modules/apps/segments/segments-web/src/main/resources/META-INF/resources/css/_contributor_builder.scss [hunk 2]
```diff
@@ -9,31 +9,15 @@
 	}
```
Comment (sandrodw3) [inline modules/apps/segments/segments-web/src/main/resources/META-INF/resources/css/_contributor_builder.scss:9]: I think we still have some CSS we can change in this file, I see some `overflow`, `height`, `position` rules that we can replace with classes 😄

File: modules/apps/segments/segments-web/src/main/resources/META-INF/resources/css/_criteria_group.scss [hunk 3]
```diff
@@ -22,15 +22,6 @@
```
Comment (sandrodw3) [inline modules/apps/segments/segments-web/src/main/resources/META-INF/resources/css/_criteria_group.scss:22]: I still see some CSS we can replace here as well 😄,  `width`, `position`, `margin` and `padding` classes can be replaced

File: modules/apps/segments/segments-web/src/main/resources/META-INF/resources/css/_criteria_sidebar.scss [hunk 4]
```diff
@@ -3,11 +3,7 @@
 .criteria-sidebar-root {
```
Comment (sandrodw3) [inline modules/apps/segments/segments-web/src/main/resources/META-INF/resources/css/_criteria_sidebar.scss:3]: Same here, we can replace some CSS, almost all `flex` rules can be replaced with classes (https://getbootstrap.com/docs/4.0/utilities/flex/), and we still have some `margin` and `padding` rules we can replace. For rules that are using `rem`, you can search for the equivalence in `px` with some "REM to px conversor". For example, `padding: 0.25rem` is the same that `padding: 4px`, and we can replace it with `p-1` 😛

File: modules/apps/segments/segments-web/src/main/resources/META-INF/resources/css/_drop_zone.scss [hunk 5]
```diff
@@ -1,8 +1,6 @@
 @import 'atlas-variables';
```
Comment (sandrodw3) [inline modules/apps/segments/segments-web/src/main/resources/META-INF/resources/css/_drop_zone.scss:1]: Same here, we can replace `align-items`, `display`, `width` or `height`

**Review by sandrodw3 — CHANGES_REQUESTED** [review]
Hey @BarbaraCabrera, nice work 😄 almost all your changes LGTM! (except two classes I'm not sure if they exist, I left comments). Apart of that, I think we still can remove more CSS, I think I left comments in all files that can be cleaned a bit more, please take a look!

**Review by sandrodw3 — APPROVED** [review]

**Comment by sandrodw3** [comment]
ci:forward

**Comment by sandrodw3** [comment]
ci:forward

**Comment by sandrodw3** [comment]
ci:forward

**Pull request closed without merging**